conftest.py

```diff
@@ -36,26 +37,32 @@ def pytest_addoption(parser):
 
 def pytest_sessionstart(session):
     workerinput = getattr(session.config, "workerinput", None)
+    should_add_lrf_data = "lrf_data or not lrf_data" in session.config.getoption("-m")
```

Comment:
Great, the only problem I see is it will not work if you will switch markers like using
"not lrf_data or lrf_data"
instead of 
"lrf_data or not lrf_data"
it should be the case. Can you please adjust?

Reply:
or "lrf_data or ontology"